CONTRIBUTING.md

```diff
@@ -4,11 +4,11 @@ Want to hack on kubernetes? Yay!
 
 ## Developer Guide
 
-We have a [Developer's Guide](docs/devel/README.md) that outlines everything you need to know from setting up your dev environment to how to get faster Pull Request reviews. And of course if you find something undocumented or incorrect along the way, feel free to send a PR!
+We have a [Developer's Guide](docs/devel/README.md) that outlines everything you need to know from setting up your dev environment to how to get faster Pull Request reviews. If you find something undocumented or incorrect along the way, please feel free to send a Pull Request.
```

Comment:
We use the phrase `PR` many places. Could/should we say `... to send a Pull Request (PR).` So they can some context for the acronym in later reading?